### ISSUE TITLE
fail to install SVL

### ISSUE THREAD

**Issue by frankie-wei** [lgsvl/simulator#1923]
I use GPU resource in alibaba cloud. I install Ubuntu 20.04（64-bit）first then install windows 10 as double system OS.

the GPU model as：GN6I 4 core 15 GB memory 1 NVIDIA T4

it encounter below error message

![error page](https://user-images.githubusercontent.com/69788063/146732047-c8738014-c9e3-4ba6-aebc-2f7d59b8b079.png)

try to install gpu drive firtst then install CUDA and cudnn and LgSVL but still fail。

![error page1](https://user-images.githubusercontent.com/69788063/146732170-0aa4ff95-8a2f-4ae1-8eca-f495abf2ac7b.png)

I also run nvidia-smi to check gpu status.

please kindly advise how to resolve the failure problem

![error page2](https://user-images.githubusercontent.com/69788063/146732270-c7e30492-6bbb-440b-addb-a9a62c1196bf.png)

**Comment by EricBoiseLGSVL** [comment]
Why are you installing windows and Linux?  To just run the simulator executable, you only need the windows or the linux application.  I recommend you use our 2021.3 release build and run on the OS for the build you have.  After that, the GPU may have issues running Unity but should be fine unless it doesn't support the vulkan drivers.

**Comment by frankie-wei** [comment]
> Why are you installing windows and Linux? To just run the simulator executable, you only need the windows or the linux application. I recommend you use our 2021.3 release build and run on the OS for the build you have. After that, the GPU may have issues running Unity but should be fine unless it doesn't support the vulkan drivers.

Eric, thanks for your prompt support. May I know if it could install 2021.3 release for linux on Ubuntu 20.04（64-bit）?

**Comment by EricBoiseLGSVL** [comment]
Yes, but I am confused on the install question.  Are you installing Unity Editor or just running the 2021.3 release binary?

**Comment by frankie-wei** [comment]
Eric, would you please help to advice the correct steps to install SVLsimulator

**Comment by EricBoiseLGSVL** [comment]
Sure but what do you mean install?  SVLSimulator can be a binary that you run on your machine or you can install the Unity Editor and clone svlsimulator repositories if you want to build custom svlsimulator binaries or custom assets.
Keep in mind we do not support running the Unity editor in the cloud, is this what you are trying to do?
